src/main/resources/com/google/api/codegen/clientconfig/json.snip

```diff
@@ -88,12 +88,12 @@
                 retryCodesName = methodConfig.getRetryCodesConfigName, \
                 retryParamsName = methodConfig.getRetrySettingsConfigName
             "{@method.getSimpleName}": {
-              @if or(and(retryCodesName, retryParamsName), isBatching)
+              @if or(and(and(retryCodesName, retryParamsName), context.isRetryingSupported(method)), isBatching)
```

Comment:
Ack I didn't realize that clientconfig generation wasn't on MVVM. Since this is totally outside the scope of this PR, can you make sure an issue is filed if one isn't yet?